llvm/docs/LangRef.rst

```diff
@@ -10012,6 +10013,12 @@ The truth table used for the '``or``' instruction is:
 |   1 |   1 |   1 |
 +-----+-----+-----+
 
+``disjoint`` means every bit is known to be zero in at least one of the inputs.
```

Comment:
My reading of "every bit" seems a bit ambiguous. I thought the point was that comparing each individual bit, each must be 0. But this reads to me more like one of the values as a whole is 0

Reply:
That was not my intention. I'll try to reword it.